sparsity/dask/core.py

```diff
@@ -149,6 +157,19 @@ def repartition(self, divisions=None, npartitions=None, force=False):
             return repartition_npartitions(self, npartitions)
         raise ValueError('Either divisions or npartitions must be supplied')
 
+    def get_partition(self, n):
+        """Get a sparse dask DataFrame/Series representing
+           the `nth` partition."""
+        if 0 <= n < self.npartitions:
+            name = 'get-partition-%s-%s' % (str(n), self._name)
+            dsk = {(name, 0): (self._name, n)}
```

Comment:
I don't get why `self._name` is used as a function here, but I see this is copied from dask, so I'm assuming it works.

Reply:
Ahhh ok this is how dask collections are managed `self._name` is not used as a function but the tuple `(self._name, n)` references the n-th partition of the collection :)